lib/cisco_node_utils/bgp.rb

```diff
@@ -416,13 +416,24 @@ def event_history_cli
       match = config_get('bgp', 'event_history_cli', @get_args)
       if match.is_a?(Array)
         return 'false' if match[0] == 'no '
-        return 'size_' + match[1] if match[1]
```

Comment:
Similar refactoring can be done for other properties below that follow the same pattern.